robot/src/main/java/org/frcteam2910/common/robot/drivers/Limelight.java

```diff
@@ -44,26 +68,50 @@ public Limelight(NetworkTable table) {
         snapshot = table.getEntry("snapshot");
     }
 
+    /*
+     * Returns true if the Limelight has a target, else false
+     * @returns A boolean value stating whether the Limelight has a target
```

Comment:
We already know the return value is a boolean

Reply:
done